postgresql/resource_postgresql_grant.go

```diff
@@ -55,6 +57,18 @@ func resourcePostgreSQLGrant() *schema.Resource {
 				ForceNew:    true,
 				Description: "The database schema to grant privileges on for this role",
 			},
+			"foreign_data_wrapper": {
```

Comment:
Now that #105 has been merged, I think we can use the `objects` settings instead of having this 2 new fields? What do you think?
(If needed you can assert that the len of `objects` if 1)